esp-hal/src/gpio/mod.rs

```diff
@@ -217,9 +227,14 @@ pub enum DriveStrength {
 ///
 /// GPIO pins can be configured for various functions, such as GPIO
 /// or being directly connected to a peripheral's signal like UART, SPI, etc.
-/// The `AlternateFunction` enum allows to select one of several functions that
+/// The `AlternateFunction` enum allows selecting one of several functions that
 /// a pin can perform, rather than using it as a general-purpose input or
 /// output.
+///
+/// The different variants correspond to different functionality depending on
+/// the chip and the specific pin. For more information, refer to your chip's
+#[doc = crate::trm_markdown_link!()]
```

Comment:
maybe the macro could optionally select the right chapter (e..g. here we would add the anchor `#iomuxgpio`)

Reply:
(not necessarily in this PR)

Reply:
I guess we can do it here, but not necessarily everywhere else if we add more TRM links

Reply:
yes - the idea was to have it if we can but not necessarily always require an anchor

Reply:
Simple enough, will do

Reply:
`iomuxgpio` doesn't seem to work for me.

Reply:
works fine in Chrome for me 🤔